FairPlay/BasicFairPlayPlayer/swift/BasicFairPlayPlayer/ViewController.swift

```diff
@@ -90,17 +95,21 @@ class ViewController: UIViewController, BCOVPlaybackControllerDelegate {
             self.fairPlayAuthProxy = BCOVFPSBrightcoveAuthProxy(publisherId: kViewControllerFairPlayPublisherId,
                                                                 applicationId: kViewControllerFairPlayApplicationId)
             
-            self.fairPlayAuthProxy?.retrieveApplicationCertificate() { (applicationCertificate: Data?, error: Error?) -> Void in
+            self.fairPlayAuthProxy?.retrieveApplicationCertificate() { [weak self] (applicationCertificate: Data?, error: Error?) -> Void in
                 guard let appCert = applicationCertificate else
                 {
                     print("ViewController Debug - Error retrieving app certificate: %@", error!)
                     return
                 }
                 
+                guard let strongSelf = self else {
```

Comment:
I won't add a :+1: to all of these sorts of improvements but I gotta say, great stuff!